examples/locks/locks.description

```diff
@@ -1,2 +1,2 @@
-// Ballerina locks are used for concurrency management, encapsulating block of statements will acquire
-// the locks for each global or service level variable reference used within those block of statements.
+// Ballerina locks are used for concurrency management. Encapsulating blocks of statements will acquire
```

Comment:
Since locks is attached to a single block, shouldn't we use "The encapsulating block of statements" here ?